mod-source-record-storage-server/src/main/java/org/folio/services/handlers/actions/MarcAuthorityDeleteEventHandler.java

```diff
@@ -0,0 +1,29 @@
+package org.folio.services.handlers.actions;
+
+import io.vertx.core.Future;
+import org.folio.rest.jaxrs.model.Record;
+import org.folio.services.RecordService;
+import org.folio.services.util.TypeConnection;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import static org.folio.rest.jaxrs.model.DataImportEventTypes.DI_SRS_MARC_AUTHORITY_RECORD_DELETED;
+
+@Component
+public class MarcAuthorityDeleteEventHandler extends AbstractDeleteEventHandler {
+
+  @Autowired
+  public MarcAuthorityDeleteEventHandler(RecordService recordService) {
+    super(recordService, TypeConnection.MARC_AUTHORITY);
+  }
+
+  @Override
+  protected Future<Boolean> deleteRecord(Record record, String tenantId) {
+    return recordService.deleteRecordById(record.getId(), tenantId);
+  }
```

Comment:
Could you please explain your decision to make this method abstract and overrode? I think that approach of deleting record will be the same for all type of records.